build.sbt

```diff
@@ -39,11 +38,11 @@ lazy val microservice = Project(appName, file("."))
   .disablePlugins(JUnitXmlReportPlugin) //Required to prevent https://github.com/scalatest/scalatest/issues/1427
   .settings(
     scalaVersion := "2.12.16",
-    SilencerSettings(),
```

Comment:
I think `SilencerSettings()` is still needed

Reply:
That is recommended here: https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=TEC&postingDay=2022%2F9%2F23&title=Scala+2.13+is+now+supported

Reply:
You can copy the [scalacOptions](https://github.com/hmrc/trusts/blob/97625baad5e5b72472274d4ca8b7dae03986b307/project/SilencerSettings.scala#L12-L15) into build.sbt and get rid of the rest of the `SilencerSettings()` file. See example commit [here](https://github.com/hmrc/internal-auth-frontend/pull/20/commits/bd9a3c08e0a90024b5c4dbc4453df74b462103a0)